packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -894,6 +908,86 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
         return dragValue;
     };
 
+    /**
+     * Handles touch dragging
+     */
+    private handleUpperThumbTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.upperThumb);
+    };
+
+    private handleLowerThumbTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.lowerThumb);
+    };
+
+    private handleThumbTouchStart = (e: React.TouchEvent, thumb: SliderThumb): void => {
+        if (e.defaultPrevented || this.isBusyOrDisabled()) {
+            return;
+        }
+
+        e.preventDefault();
+        (e.target as HTMLElement).focus();
+        window.addEventListener("touchend", this.handleTouchEnd);
```

Comment:
You should be sure to remove these listeners on unmount to ensure they aren't called and causing an unmounted component to setState.

Reply:
We call "suspendActiveOperations" on unmount which shuts down all dragging/incrementing listeners.

Reply:
awesome - must have missed that. Thanks